core-plugins/src/e2e-test/features/groupby/GroupByWithFile.feature

```diff
@@ -153,3 +153,156 @@ Feature: GroupBy - Verify File source to File sink data transfer using GroupBy a
+    When Select plugin: "Group By" from the plugins list as: "Analytics"
+    Then Connect plugins: "File" and "Group By" to establish connection
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "File" from the plugins list as: "Sink"
+    Then Connect plugins: "Group By" and "File2" to establish connection
+    Then Navigate to the properties page of plugin: "File"
+    Then Enter input plugin property: "referenceName" with value: "FileReferenceName"
+    Then Enter input plugin property: "path" with value: "groupByTest"
+    Then Select dropdown plugin property: "format" with option value: "csv"
+    Then Click plugin property: "skipHeader"
+    Then Click on the Get Schema button
+    Then Verify the Output Schema matches the Expected Schema: "groupByCsvDataTypeFileSchema"
+    Then Validate "File" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "Group By"
+    Then Select dropdown plugin property: "groupByFields" with option value: "groupByValidFirstField"
+    Then Press ESC key to close the unique fields dropdown
+    Then Select dropdown plugin property: "groupByFields" with option value: "groupByValidSecondField"
+    Then Press ESC key to close the unique fields dropdown
+    Then Enter GroupBy plugin Fields to be Aggregate "groupByFileAggregateMultipleSetFields1"
+    Then Click on the Get Schema button
+    Then Click on the Validate button
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "File2"
+    Then Enter input plugin property: "referenceName" with value: "FileReferenceName"
+    Then Enter input plugin property: "path" with value: "fileSinkTargetBucket"
+    Then Replace input plugin property: "pathSuffix" with value: "yyyy-MM-dd-HH-mm-ss"
+    Then Select dropdown plugin property: "format" with option value: "tsv"
+    Then Validate "File2" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Close the pipeline logs
+    Then Validate OUT record count of groupby is equal to IN record count of sink
+    Then Validate output file generated by file sink plugin "fileSinkTargetBucket" is equal to expected output file "groupByTest5OutputFile"
+
+  @GROUP_BY_TEST @FILE_SINK_TEST
+  Scenario: To verify complete flow of data extract and transfer from File source to File sink with GroupBy plugin using set of aggregates
```

Comment:
nit: update scenario description to mention which aggregates are being tested (count, sum, max)

Reply:
Updated scenario description.